lte/gateway/c/session_manager/SessionStateEnforcer.cpp

```diff
@@ -283,6 +283,7 @@ void SessionStateEnforcer::m5g_start_session_termination(
     const uint32_t& pdu_id, SessionStateUpdateCriteria* session_uc) {
   const auto session_id   = session->get_session_id();
   const std::string& imsi = session->get_imsi();
+  const auto previous_state = session->get_state();

```

Comment:
why is this labeled as "previous" state?

Reply:
As part of m5g_start_session_termination() definition first step we are doing is changing the state to SESSION_RELEASED and then updating AMF, removing rules from UPF,...etc please find the below screenshot for reference.
![image](https://user-images.githubusercontent.com/70899516/134646441-61bae1fd-e09b-49e9-b98a-9afc59a53c88.png)

So, before we perform removing rules from UPF we need to check the session state by the time m5g_start_session_termination() called. so created a local variable and stored the session state by the time m5g_start_session_termination() called.

If you thinking this identifier "previous_state" not good for readability, please suggest me some name I can change it.